src/transforms/select.jl

```diff
@@ -62,12 +94,14 @@ function apply(transform::Select, table)
   rcols = [Tables.getcolumn(cols, name) for name in reject]
 
   # table with selected columns
-  𝒯 = (; zip(select, scols)...)
-  stable = 𝒯 |> Tables.materializer(table)
+  # 𝒯 = (; zip(select, scols)...)
+  # stable = 𝒯 |> Tables.materializer(table)
 
-  stable, (reject, rcols, sperm, rinds)
+  TableSelection(table, select), (reject, rcols, sperm, rinds)
 end
 
+revert(::Select, newtable::TableSelection, cache) = newtable.table
+
```

Comment:
Do we need to add a method for TableSelection? I thought that we would leave everything else untouched because TableSelection is a valid Tables.jl table.

Reply:
This method is used for a faster revert function call.